articles/service-bus-messaging/build-message-driven-apps-nservicebus.md

```diff
@@ -32,10 +32,10 @@ The sample assumes you've [created an Azure Service Bus namespace](service-bus-q
     The following diagram, generated by [ServiceInsight](https://particular.net/serviceinsight), a visualization and debugging tool from Particular Software, shows the message flow:
     
     :::image type="content" source="./media/nservicebus/sequence-diagram.png" alt-text="Image showing the sequence diagram":::    
-1. Open `SendReceiveWithNservicebus.sln` in your favorite code editor (For example, Visual Studio 2019). 
+1. Open `SendReceiveWithNservicebus.sln` in your favorite code editor (For example, Visual Studio 2022). 
 1. Open `appsettings.json` in both the Receiver and Sender projects and set `AzureServiceBusConnectionString` to the connection string for your Azure Service Bus namespace.
-
-
+   - This can be found in the Azure portal under the Service Bus Namespace -> Settings -> Shared access policies -> RootManageSharedAccessKey -> Primary Connection String
+   - The `AzureServiceBusTransport` also has a constructor that accepts a namespace and token credential, which in a production environment will be more secure, however for the purposes of this tutorial the shared access key connection string will be used
```

Comment:
```suggestion
   - This can be found in the Azure portal under  **Service Bus Namespace** > **Settings** > **Shared access policies** > **RootManageSharedAccessKey** > **Primary Connection String** .
   - The `AzureServiceBusTransport` also has a constructor that accepts a namespace and token credential, which in a production environment will be more secure, however for the purposes of this tutorial the shared access key connection string will be used.
```
fixing syntax for MS Style